tests/E2E.test.ts

```diff
@@ -6,7 +6,11 @@ import {
   getLanguageChunk,
 } from '@vocab-private/test-helpers';
 
-describe('E2E', () => {
+const describeWithTimeout = (title: string, fn: () => any) =>
+  // eslint-disable-next-line jest/valid-describe-callback
+  describe.each([title])('%s', fn, 2000);
```

Comment:
Is there much point in setting this since the default jest timeout is 5 seconds?

Reply:
> Waiting failed: 500ms exceeded
https://github.com/seek-oss/vocab/actions/runs/4804725133/jobs/8550580860#step:10:188

Reply:
This is very weird.
https://github.com/seek-oss/vocab/blob/master/jest.setup.js#L3

Reply:
I think the problem is calling `jest.setTimeout` outside of a test [`342da44` (#143)](https://github.com/seek-oss/vocab/pull/143/commits/342da44c682aac649f669a8e5a85a9ed66cad37c)

Reply:
I think that worked:
- https://github.com/seek-oss/vocab/actions/runs/4804916802/jobs/8550846336#step:10:170
- https://github.com/seek-oss/vocab/actions/runs/4804916802/jobs/8550846336#step:10:166

Reply:
Or not 😢 https://github.com/seek-oss/vocab/actions/runs/4804931476/jobs/8550873187#step:10:188

Reply:
![crazy times](https://media.tenor.com/CzFHocLd8SUAAAAC/rollercoaster-of-emotions-rollercoaster.gif)

Reply:
Yeah, because it wasn't Jest that was timing out; it was Puppeteer.

Reply:
I did a few reruns and it seems to hold up, so I'm merging it